src/services/terms.ts

```diff
@@ -1,20 +1,37 @@
 import * as RDF from 'rdf-js';
 
 export class Term {
-  id: RDF.Term | undefined = undefined;
+  constructor(
+    readonly id: RDF.Term,
+    readonly type: RDF.Term | undefined,
+    readonly prefLabels: RDF.Term[],
+    readonly altLabels: RDF.Term[],
+    readonly hiddenLabels: RDF.Term[],
+    readonly scopeNotes: RDF.Term[],
+    readonly broader: RelatedTerm[],
```

Comment:
'prefLabels', 'altLabels', etc, are in plural, but 'broader' and 'narrower' are not. Does it make sense to rename these to e.g. 'broaderTerms' and 'narrowerTerms'?

Reply:
It sure does! As we have simple, singular words in the GraphQL schema (`altLabel` etc.) I decided to keep `broader`/`narrower` there.